contracts/V1/StateChannelDiamondProxy/DisputeManagerFacet.sol

```diff
@@ -49,31 +49,33 @@ contract DisputeManagerFacet is StateChannelCommon {
         require(!_isExpired(timestamp), ErrorDisputeExpired());
         require(_isCorrectGenesis(dispute,disputeAuditingData), ErrorDisputeGenesisInvalid());
         require(_verifyStateProof(dispute, disputeAuditingData), ErrorDisputeStateProofInvalid());
-        //TODO! Verify exitChannelBlocks
+        require(_verifyJoinChannelBlocks(dispute, disputeAuditingData), ErrorDisputeJoinChannelBlocksInvalid());
+        require(_verifyExitChannelBlocks(dispute, disputeAuditingData), ErrorDisputeExitChannelBlocksInvalid());
 
         FraudProofVerificationContext memory poofContext = FraudProofVerificationContext({
             channelId: dispute.channelId
         }); 
-        (bytes memory encodedModifiedState, ExitChannel[] memory exitChannels) = playDisputeOutputGeneration(
+        (bytes memory encodedModifiedState, ExitChannelBlock memory eBlock, 
+        Balance memory totalDeposits, Balance memory totalWithdrawals) = playDisputeOutputGeneration(
```

Comment:
lets change the name now, so we don't make it canonical

Reply:
changed it to generateDisputeOutputState